src/main/java/de/uhd/ifi/se/decision/management/jira/filtering/impl/FilterExtractorImpl.java

```diff
@@ -221,20 +220,17 @@ private boolean checkIfKnowledgeTypeMatches(DecisionKnowledgeElement element) {
 			if (filterSettings.getDocumentationLocations().contains(element.getDocumentationLocation())
 					|| filterSettings.getDocumentationLocations().size() == 1 && filterSettings
 							.getDocumentationLocations().get(0).equals(DocumentationLocation.UNKNOWN)) {
-				// Check if the Status is filtered
-				if (filterSettings.getSelectedStatus()
-						.contains(StatusPersistenceManager.getStatusForElement(element))) {
-					// Check if the Type of the Element is correct
-					if (checkIfKnowledgeTypeMatches(element) && checkIfElementMatchesTimeFilter(element)) {
-						// Case no text filter
-						if (filterSettings.getSearchString().equals("")
-								|| filterSettings.getSearchString().equals("?filter=-4")
-								|| filterSettings.getSearchString().equals("?filter=allopenissues")) {
+				// Check if the Status is filtered, check if the Type of the Element is correct
+				if (filterSettings.getSelectedStatus().contains(element.getStatus())

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [These nested if statements could be combined](https://app.codacy.com/app/UHD/cures-condec-jira/pullRequest?prid=4636195)